libs/sdk-core/src/persist/migrations.rs

```diff
@@ -460,5 +460,6 @@ pub(crate) fn current_sync_migrations() -> Vec<&'static str> {
         channel_opening_fees TEXT NOT NULL
        ) STRICT;
        ",
+        "ALTER TABLE payments_external_info ADD COLUMN lnurl_withdraw_endpoint TEXT;",
```

Comment:
Since we already had a migration in main db this statement might have already been running so it will fail bow with column already exists.
In sqlite for these cases we need to create a temp table with the right scheme copy the data to the temp table, remove the originall table and as a final step rename the temp table to the original name

Reply:
We have an example in our code

Reply:
Its not the case here, because the error I listed above is "no such table: payments_external_info"

Reply:
I ran the CLI on this branch with a new wallet (fresh invite code, no local SQL files) and got the same error:

```
sdk> connect -i "<invite-code>"

Error: Failed to use the local DB for persistence:
  rusqlite_migrate error: RusqliteError {
    query: "ALTER TABLE payments_external_info ADD COLUMN lnurl_withdraw_endpoint TEXT;",
    err: SqliteFailure(
      Error {code: Unknown, extended_code: 1 },
      Some("no such table: payments_external_info")
    )
  }
```